docs/installation/mac.md

```diff
@@ -107,25 +107,6 @@ To start elasticsearch as a daemonized process:
 ./bin/elasticsearch -d
 ```
 
-### Installing Elasticsearch with Homebrew
```

Comment:
If we switch to "it runs on 7.5 or higher" we could in theory keep these instructions, after all the official docs still support ES via Homebrew: https://www.elastic.co/guide/en/elasticsearch/reference/current/brew.html

(I'm using `elasticsearch-oss` 7.9.1 via `brew tap elastic/tap` in development)

The risk is that someone will install ES 8 when it comes out and maybe it won't work so I'm also in favor of scrapping them.

If we keep them I'd replace them with something like:

```
brew tap elastic/tap
brew install elastic/tap/elasticsearch-oss
brew pin elasticsearch-oss
```

The rest of the post installation instructions would be the same.

Reply:
Given my recent conversations with @mstruve and @jdoss, it seems like "unofficially" supporting 7.5 and above is a good step. I'd MUCH rather keep in the homebrew installation instructions if we can!

(@cmgorton and I went 10 rounds with `brew` yesterday and there's just no good/easy way to install older versions of Elasticsearch anymore)

Reply:
I'm cool with bumping it to 7.x version that works.

Reply:
Ok, maybe I should close this PR for now and create a Draft PR until we all come to a consensus on brew and Elasticsearch. Thoughts?

Reply:
I'd be comfortable just changing this PR to a) mentioned we use Elasticsearch 7.5 and up, and b) changing the home-brew instructions to pull right from the latest version like @rhymes mentioned at https://github.com/forem/forem/pull/10370#discussion_r490709809!

Reply:
Hey I am circling back on this as I know I let it lapse. I had a community member who was going to pair with Nick this week unable to get to get the installation working because of elasticsearch. Should I recommend he pull from the latest version like @rhymes and @joshpuetz mentioned?

If we are still ok with that I can also update this PR to do the same.

Reply:
I'd vote for specifying the latest version of Elasticsearch!

Reply:
Any 7.x version of Elasticsearch will work and can be recommended